src/main/java/br/com/labbs/monitor/filter/MetricsCollectorFilter.java

```diff
@@ -57,7 +58,14 @@
     private static final String ERROR_MESSAGE_PARAM = "error-message";
     private static final String DEBUG = "debug";
     private static final String APPLICATION_VERSION = "application-version";
+    private static final String DEFAULT_FILTER_REGEX = "[^A-zÀ-ú .,]+";
```

Comment:
Check if " " is accepted. It would be better to use "\s" for representing spaces in regex.